src/engraving/libmscore/mscore.cpp

```diff
@@ -134,8 +133,6 @@ void MScore::init()
     playRepeats         = true;
     playbackSpeedIncrement = 5;
 
-    lastError           = u"";
```

Comment:
I looked, at the moment this error string was shown to the user literally in one place (when saving the style file), in other cases, it was simply output to the log.

We have two problems with this:
1. Output of translated strings to the log is very inconvenient for developers to read logs (for example, from Chinese)
2. This message may be formed incorrectly or not fully or too technically - this information may be redundant and useless for the user (poor context)

It is better to return an error code or a type of Ret, and form a message for the user closer to the place of the call, where there is a full context about what is happening.

Also, using this static string for errors is bad from a technical point of view - it will become a problem if (when) we call the engraving methods in different threads.

Reply:
Yes, I see that it is a bad solution, and in fact I'm happy to see it being removed :)
What I meant was that it was used for UI purposes much more often in MU3 than in MU4. So as long as we don't forget to create a new solution for informing the user about errors where necessary, I'm totally happy with this removal!